refinery/tool_manager/models.py

```diff
@@ -259,16 +269,53 @@ def get_relative_container_url(self):
         """
         Construct & return the relative url of our Tool's container
         """
-        return "/visualizations/{}".format(self.container_name)
+        return "/{}/{}".format(
+            settings.DJANGO_DOCKER_ENGINE_BASE_URL,
+            self.container_name
+        )
 
     def get_tool_name(self):
         return self.tool_definition.name
 
     def get_tool_type(self):
         return self.tool_definition.tool_type
 
-    def parse_file_relationships_string(self):
-        raise NotImplementedError
+    def update_file_relationships_string(self):
+        """
+        Replace a Tool's Node uuids in its `file_relationships` string with
+        their respective FileStoreItem's urls. No error handling here since
+        this method is only called in an atomic transaction.
+        """
+        node_uuids = re.findall(
+            r'[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}',
+            self.file_relationships
+        )
 
-    def populate_url_from_node_uuid(self):
+        for uuid in node_uuids:
```

Comment:
I think the functionality of getting a file URL from a Node UUID could be useful elsewhere down the road. Not necessary strictly speaking though.